lib/robot.js

```diff
@@ -83,20 +86,24 @@ class Robot {
    *   // An issue was just opened.
    * });
    */
-  on (event, callback) {
-    if (event.constructor === Array) {
-      event.forEach(e => this.on(e, callback))
+  on (webhook, callback) {
+    if (webhook.constructor === Array) {
+      webhook.forEach(e => this.on(e, callback))
```

Comment:
Is there a reason for renaming this variable?

Reply:
I had to rename this variable because `event` was colliding in line 97 (`return this.events.on(name, async event => { … })`) and I still needed to access the first `event` to check if it's marked as a `breakingEvent`.